src/backy/main.py

```diff
@@ -44,10 +34,10 @@ def __init__(self, path, log):
         self.path = path
         self.log = log
 
-    def status(self, yaml_: bool):
-        b = backy.backup.Backup(self.path, self.log)
+    def status(self, yaml_: bool, revision):
+        revs = backy.backup.Backup(self.path, self.log).find_revisions(revision)
         if yaml_:
-            print(yaml.safe_dump([r.to_dict() for r in b.clean_history]))
+            print(yaml.safe_dump([r.to_dict() for r in revs]))
```

Comment:
verify that the directory accepts revisions that are incomplete or at least gracefully ignores them